src/Console/Commands/BassetNamedAssetsList.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Backpack\Basset\Console\Commands;
+
+use Backpack\Basset\Facades\Basset;
+use Illuminate\Console\Command;
+use Illuminate\Support\Str;
+
+/**
+ * Basset Named Assets list command.
+ *
+ * @property object $output
+ */
+class BassetNamedAssetsList extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'basset:named-assets {--filter=*}';
```

Comment:
I expect a command to be a verb. Because a command DOES SOMETHING. If this command _lists_ the assets that the packages have added to the map... why not call it `php artisan basset:list`?

```suggestion
    protected $signature = 'basset:list {--filter=*}';
```